docusaurus/docs/reactnative/basics/getting_started.mdx

```diff
@@ -93,7 +93,7 @@ To be able to use the Stream Chat React Native SDK, a few dependencies must meet
 
 | react-native | stream-chat-react-native (min required) | react-native-reanimated (min required) |
 | ------------ | --------------------------------------- | -------------------------------------- |
-| 0.67         | 4.2.0                                   | 2.4.0                                  |
+| 0.67         | 4.2.0                                   | 2.7.0                                  |
```

Comment:
This is not true.. its more like only 4.11.0 will need 2.7.0

Reply:
also reanimated 2.6 adds support of RN 0.68, so lets add a new row and keep the previous one the same

0.68 - 4.11.0 - 2.7.0

I assume 4.11 gets triggered after we merge this

Reply:
Kinda confused if we should add for react native 0.68, but the other two make complete sense.